### PR TITLE
RangeSlider: update demo description

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: JSDemos/Demos/RangeSlider/Overview/Angular/description.md [hunk 1]
```diff
@@ -28,8 +28,10 @@ The RangeSlider can also display a tooltip for slider handles. To display toolti
...
\ No newline at end of file
+To handle value changes, use two-way binding to bind the [value](/Documentation/ApiReference/UI_Components/dxNumberBox/Configuration/#value) properties of [NumberBox](/Documentation/ApiReference/UI_Components/dxNumberBox/) components to the [start](/Documentation/ApiReference/UI_Components/dxRangeSlider/Configuration/#start) and [end](/Documentation/ApiReference/UI_Components/dxRangeSlider/Configuration/#end) properties of the RangeSlider.
+
+The [callValueChange](/Documentation/ApiReference/UI_Components/dxRangeSlider/Configuration/#callValueChange) property allows you to choose when to change the RangeSlider value and supports two modes: `onMoving` and `onMovingComplete`. You can see the property effect in the **Process Value Changes** section. The first RangeSlider changes its value every time a user slides the handle. The second RangeSlider changes its value only on handle release.
```
Comment (dirk-pieterse) [inline JSDemos/Demos/RangeSlider/Overview/Angular/description.md]:
```suggestion
The [callValueChange](/Documentation/ApiReference/UI_Components/dxRangeSlider/Configuration/#callValueChange) property allows you to choose when to change the RangeSlider value and supports the `onMoving` and `onMovingComplete` modes. You can see the property's effect in the **Process Value Changes** section. The first RangeSlider changes its value every time a user slides the handle. The second RangeSlider changes its value only when the handle is released.
```

File: JSDemos/Demos/RangeSlider/Overview/Vue/description.md [hunk 2]
```diff
@@ -28,8 +28,10 @@ The RangeSlider can also display a tooltip for slider handles. To display toolti
...
\ No newline at end of file
+To handle value changes, use two-way binding to bind the [value](/Documentation/ApiReference/UI_Components/dxRangeSlider/Configuration/#value) properties of different components. In this demo, you can use the [NumberBox](/Documentation/ApiReference/UI_Components/dxNumberBox/) or RangeSlider component to change the range because their **value** properties are bound.
+
+The [callValueChange](/Documentation/ApiReference/UI_Components/dxRangeSlider/Configuration/#callValueChange) property allows you to choose when to change the RangeSlider value and supports two modes: `onMoving` and `onMovingComplete`. You can see the property effect in the **Process Value Changes** section. The first RangeSlider changes its value every time a user slides the handle. The second RangeSlider changes its value only on handle release.
```
Comment (dirk-pieterse) [inline JSDemos/Demos/RangeSlider/Overview/Vue/description.md]: Same here

File: JSDemos/Demos/RangeSlider/Overview/description.md [hunk 3]
```diff
@@ -28,8 +28,10 @@ The RangeSlider can also display a tooltip for slider handles. To display toolti
...
 
-Specify the [onValueChanged](/Documentation/ApiReference/UI_Components/dxRangeSlider/Configuration/#onValueChanged) function to handle the **value** change. In this demo, the [NumberBox](/Documentation/ApiReference/UI_Components/dxNumberBox/) components and the RangeSlider use this property to exchange values. You can use one of these components to change the range.
\ No newline at end of file
+Specify the [onValueChanged](/Documentation/ApiReference/UI_Components/dxRangeSlider/Configuration/#onValueChanged) function to handle the **value** change. In this demo, the [NumberBox](/Documentation/ApiReference/UI_Components/dxNumberBox/) components and the RangeSlider use this property to exchange values.
+
+The [callValueChange](/Documentation/ApiReference/UI_Components/dxRangeSlider/Configuration/#callValueChange) property allows you to choose when to change the RangeSlider value and supports two modes: `onMoving` and `onMovingComplete`. You can see the property effect in the **Process Value Changes** section. The first RangeSlider changes its value every time a user slides the handle. The second RangeSlider changes its value only on handle release.
```
Comment (dirk-pieterse) [inline JSDemos/Demos/RangeSlider/Overview/description.md]: and here

**Review by dirk-pieterse — DISMISSED** [review]

File: JSDemos/Demos/RangeSlider/Overview/Vue/description.md (hunk 2 above)
Comment (vladaskorohodova) [inline JSDemos/Demos/RangeSlider/Overview/Vue/description.md]:
```suggestion
The [callValueChange](/Documentation/ApiReference/UI_Components/dxRangeSlider/Configuration/#callValueChange) property allows you to choose when to change the RangeSlider value and supports the `onMoving` and `onMovingComplete` modes. You can see the property's effect in the **Process Value Changes** section. The first RangeSlider changes its value every time a user slides the handle. The second RangeSlider changes its value only when the handle is released.
```

File: JSDemos/Demos/RangeSlider/Overview/description.md (hunk 3 above)
Comment (vladaskorohodova) [inline JSDemos/Demos/RangeSlider/Overview/description.md]:
```suggestion
The [callValueChange](/Documentation/ApiReference/UI_Components/dxRangeSlider/Configuration/#callValueChange) property allows you to choose when to change the RangeSlider value and supports the `onMoving` and `onMovingComplete` modes. You can see the property's effect in the **Process Value Changes** section. The first RangeSlider changes its value every time a user slides the handle. The second RangeSlider changes its value only when the handle is released.
```

File: JSDemos/Demos/RangeSlider/Overview/Angular/description.md (hunk 1 above)
Comment (albertov05) [inline JSDemos/Demos/RangeSlider/Overview/Angular/description.md]:
```suggestion
The [callValueChange](/Documentation/ApiReference/UI_Components/dxRangeSlider/Configuration/#callValueChange) property allows you to choose when to change the RangeSlider value and supports two modes: `onMoving` and `onMovingComplete`. You can see the property's effect in the **Process Value Changes** section. The first RangeSlider changes its value every time a user slides the handle. The second RangeSlider changes its value only on handle release.
```

File: JSDemos/Demos/RangeSlider/Overview/Vue/description.md (hunk 2 above)
Comment (albertov05) [inline JSDemos/Demos/RangeSlider/Overview/Vue/description.md]:
```suggestion
The [callValueChange](/Documentation/ApiReference/UI_Components/dxRangeSlider/Configuration/#callValueChange) property allows you to choose when to change the RangeSlider value and supports two modes: `onMoving` and `onMovingComplete`. You can see the property's effect in the **Process Value Changes** section. The first RangeSlider changes its value every time a user slides the handle. The second RangeSlider changes its value only on handle release.
```

File: JSDemos/Demos/RangeSlider/Overview/description.md (hunk 3 above)
Comment (albertov05) [inline JSDemos/Demos/RangeSlider/Overview/description.md]:
```suggestion
The [callValueChange](/Documentation/ApiReference/UI_Components/dxRangeSlider/Configuration/#callValueChange) property allows you to choose when to change the RangeSlider value and supports two modes: `onMoving` and `onMovingComplete`. You can see the property's effect in the **Process Value Changes** section. The first RangeSlider changes its value every time a user slides the handle. The second RangeSlider changes its value only on handle release.
```

**Pull request merged**